src/apis/index.js

```diff
@@ -45,15 +45,15 @@ export function setUserSettings(payload) {
 }
 
 export function removeUserDevice(payload) {
```

Comment:
Looks good. ✨ 

How about `removeUserDevice({ macId })`

Reply:
@anshumanv Can be done but thought of keeping the same structure throughout the page.